incubator/hnc/pkg/kubectl/root.go

```diff
@@ -90,6 +90,18 @@ func homeDir() string {
 	}
 	return os.Getenv("USERPROFILE") // windows
 }
+func getAllNamespaces() []string {
```

Comment:
IMO let's put this in tree.go until it's needed somewhere else?

Reply:
Actually I am putting this function inside root.go because this function uses a kubernetes client that is already created in root.go file.